src/calculateRentalCost.js

```diff
@@ -28,19 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  const rentalPrice = 40;
+  const rentalDayPrice = 40;
   const mediumDiscount = 20;
   const bigDiscount = 50;
 
   if (days < 3) {
-    return days * rentalPrice;
+    return days * rentalDayPrice;
   }
 
   if (days >= 3 && days < 7) {
-    return (days * rentalPrice) - mediumDiscount;
+    return (days * rentalDayPrice) - mediumDiscount;
   }
```

Comment:
It would be easier to check from the top, so you don't need two conditions for the second `if`.
example 
`if ( days >= 7) {return}
if (days >= 3){return}
return`